cosmos/carbon.json

```diff
@@ -326,9 +326,9 @@
       "coinDecimals": 18,
       "coinGeckoId": "usd-coin",
       "gasPriceStep": {
-        "low": 76923076923.07692,
-        "average": 96153846153.84616,
-        "high": 115384615384.61539
+        "low": 33333333333.333333333333,
+        "average": 33333333333.333333333333,
+        "high": 33333333333.333333333333
```

Comment:
Same issue as comment on L104, need to round ceil, if not user will be paying insufficient gas. Also, follow the original number, round to 5 d.p.